Lib/test/test_inspect/test_inspect.py

```diff
@@ -4051,7 +4093,7 @@ class C:
 
         with self.subTest('partial'):
             class C:
-                __call__ = functools.partial(lambda x, a: (x, a), 2)
+                __call__ = staticmethod(functools.partial(lambda x, a: (x, a), 2))
```

Comment:
BTW, this is not correct change. It invalidates the purpose of the test. The correct change is to leave a `partial` here, but change the expected result.